bin/node-template/runtime/Cargo.toml

```diff
@@ -16,7 +16,7 @@ targets = ["x86_64-unknown-linux-gnu"]
 codec = { package = "parity-scale-codec", version = "3.6.1", default-features = false, features = ["derive"] }
 scale-info = { version = "2.5.0", default-features = false, features = ["derive"] }
 
-pallet-aura = { version = "4.0.0-dev", default-features = false, path = "../../../frame/aura" }
+pallet-aura = { version = "4.0.0-dev", default-features = false, features = ["explicit-slot-duration"], path = "../../../frame/aura" }
```

Comment:
fwiw if this is more the case of something that is experimental now but you see being useful for everyone, there is also `feature = experimental` already used in a few `frame` based crates.

Reply:
This is exactly what I was looking for. Thanks